helpers/controllers/authController.ts

```diff
@@ -26,35 +26,33 @@ export const login = async (
   arg: Login,
   ctx: { req: Request }
 ) => {
-  const {
-    req: { session }
-  } = ctx
-  const { username, password } = arg
+  try {
```

Comment:
Why do you wrap this in a `try...catch` when the catch statement just re-throws the error?

Reply:
If there's any special case errors from `Sequelize` or `bcrypt` won't propagate properly to `ApolloServer` if it doesn't have a catch.

Reply:
@coltonehrman not catching will cause nodeJS to `console.warn` that we are not handling error when there are rejections. We probably don't want that if we can help it.